test/example-types/results.js

```diff
@@ -194,4 +196,23 @@ describe('results', () => {
       })
     })
   })
+  describe('rowsToObjectConverter', () => {
+    it('should convert array of objects to a single object based on populate config prop "singularObject"', () => {
+      let results = [
+        { other: { a: 1 }, user: [{ name: 'A' }, { name: 'A' }] },
+        { user: [{ name: 'B' }, { name: 'B' }] },
```

Comment:
These examples seem fairly contrived given the repetition of the `name` value for each element.